examples/src/main/scala/org/apache/spark/examples/mllib/LatentDirichletAllocationExample.scala

```diff
@@ -46,7 +46,10 @@ object LatentDirichletAllocationExample {
     val topics = ldaModel.topicsMatrix
     for (topic <- Range(0, 3)) {
       print(s"Topic $topic :")
-      for (word <- Range(0, ldaModel.vocabSize)) { print(s" ${topics(word, topic)}") }
+      for (word <- Range(0, ldaModel.vocabSize))
+          {
```

Comment:
No, we put the open brace on the preceding line with for, and don't triple indent. See any other for loop in the code.

Reply:
@srowen sure done.